examples/sgemm/SGEMM_main.cpp

```diff
@@ -231,7 +232,18 @@ int main(int argc, char **argv) {
     } else {
         printf("%s\n", argv[0]);
         printf("\nInvalid number of inputs\n");
-        getchar();
+        exit(-1);
+    }
+
+    int programCount = SGEMM_get_program_count();
+    int tileSize = SGEMM_get_tile_size();
+    if (K % programCount != 0 || K % tileSize != 0) {
+        printf("\nNumber of columns in Matrix B must be a multiple of %d and %d!\n", programCount, tileSize);
+        exit(-1);
+    }
+
+    if (M % programCount != 0) {
+        printf("\nNumber of rows in Matrix A must be a multiple of %d!\n", programCount);
```

Comment:
Same here:
``"... of %d (target width)!\n"``

Reply:
Good point, fixed